misc/acrn-config/xmls/config-xmls/nuc6cayh/industry_launch_1uos_vxworks.xml

```diff
@@ -1,5 +1,5 @@
 <acrn-config board="nuc6cayh" scenario="industry" uos_launcher="1">
-    <uos id="3">
+    <uos id="1">
```

Comment:
why it is 1?

Reply:
uos id 3 is removed from industry scenario, and 2 is used for hard rt vm.